tests/logging/jwtUtil.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/logging/jwtUtil.js
```

Comment:
Do you think we should have this file in an independent utils folder instead of logging folder?